tests/UI/campaigns/functional/BO/02_orders/05_shoppingCarts/05_deleteAbandonedCarts.js

```diff
@@ -0,0 +1,109 @@
+require('module-alias/register');
+
+// Import expect from chai
+const {expect} = require('chai');
+
+// Import utils
+import helper from '@utils/helpers';
+
+// Import test context
+import testContext from '@utils/testContext';
```

Comment:
```suggestion
// Import utils
import helper from '@utils/helpers';
import testContext from '@utils/testContext';

require('module-alias/register');

// Import expect from chai
const {expect} = require('chai');
```

you should move it before requires

Reply:
Done